app/views/content/train-to-be-a-teacher/who-do-you-want-to-teach.md

```diff
@@ -3,114 +3,85 @@ title: Who do you want to teach
 heading: Who do you want to teach
 description: |-
   Explore who’d you’d like to teach and the qualifications you need to work with early years, further education, and pupils with special educational needs.
-related_content:
-    Get support training to teach if you're disabled : "/funding-and-support/if-youre-disabled"
 promo_content:
     - content/train-to-be-a-teacher/promos/mailing-list-promo-who-do-you-want-to-teach
+calls_to_action:
+    get-school-experience:
+      name: simple
+      arguments:
+        icon: "icon-arrow"
+        text: |-
+          Use our service to search for and organise a placement in England.
+        link_text: "Get school experience"
+        link_target: "https://schoolexperience.education.gov.uk/"
 ---
 
-If you want to teach a particular student group you might need to do specific training. Find out what you can do to get the right qualifications.
+It’s good to consider what kind of teacher you want to be based on your interests, skills and experience. You might need specific training or qualifications in some areas. This page explains more about the different options.
 
-You could teach:
+## Teach in a primary or secondary school
 
-- primary (ages 5 to 11)
-- secondary (ages 11 to 18)
-- disabled students and students with special needs
-- early years (children up to the age of 5)
-- further education (for example, teaching at further education colleges or sixth form colleges)
+The school system in England is split into age groups:
 
-If you want to [teach in primary or secondary schools](/train-to-be-a-teacher/if-you-have-a-degree) there are a number of ways to get [qualified teacher status](/what-is-qts) (QTS) which you will need to teach in the majority of schools in
-England.
+### Primary (ages 5 to 11)
+Children usually start primary school at 4 years old and this phase is called Reception. This is the final stage of the Early Years Foundation Stage (EYFS).
 
-You will need a degree to teach all student groups except those in further education, depending on the type of further education you are teaching.
+There are a further 2 phases, known as key stages: 
 
-## Teach disabled pupils and pupils with special educational needs
+* key stage 1 covers year 1 and 2, up to 7 years old
```

Comment:
Should we be a bit more consistent with how we write these bullets in the primary and secondary sections?

eg include KS1 and KS2 notation?

eg make sure all the bullets are worded consistently re years and ages? for example

* key stage x (KSx) covers years x to x with children aged x to x (changed to pupils/students when appropriate)